Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM debian:10.4-slim
+
+# Install libraries
+RUN apt update && \
+    apt-get install -y curl unzip && \
+    apt-get install -y --no-install-recommends socat
+
+# Create config and data dirs
+RUN mkdir config && \
+    mkdir antfarm-data
+
+# Download sia-antfarm and siad-dev binaries
+ARG SIA_ANTFARM_VERSION=docker-test-007
+WORKDIR /sia-antfarm
+RUN curl -o tag-page.html --fail "https://gitlab.com/NebulousLabs/Sia-Ant-Farm/-/tags/${SIA_ANTFARM_VERSION}" && \
+    download_link="https://gitlab.com$(cat tag-page.html | grep job=build | grep -Po '(?<=href=\")[^\"]*')" && \
+    curl -o binaries.zip -L --fail "${download_link}" && \
+    unzip binaries.zip && \
+    mv artifacts/sia* . && \
+    rm tag-page.html && \
+    rm binaries.zip && \
+    rm -r artifacts
+
+# Copy default config
+ENV CONFIG=config/basic-renter-5-hosts-docker.json
+COPY config/basic-renter-5-hosts-docker.json config/
```

Comment:
Shouldn't this be `COPY $CONFIG config/`?

Reply:
Updated.